Dockerfile

```diff
@@ -13,4 +13,4 @@ COPY server/target/compiler/*.jar ${VERTICLE_HOME}/compiler/
 # Expose this port locally in the container.
 EXPOSE 8081
 
-ENV JAVA_OPTIONS "--module-path=compiler/ --upgrade-module-path=compiler/compiler.jar:compiler/compiler-management.jar -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI"
+ENV JAVA_OPTIONS "--module-path=compiler/ --upgrade-module-path=compiler/regex.jar:compiler/compiler.jar:compiler/compiler-management.jar -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI"
```

Comment:
@adamdickmeiss I don't think we need it on the upgrade-module-path, only on regular one.